src/peeker/config.toml

```diff
@@ -29,9 +29,7 @@ names = [
     # make these materialized once we support materialized sources
     "history",
     "district",
-    "nation",
     "region",
-    "supplier",
```

Comment:
this seems like a non-negligible change change to our nightly load tests. Since these are not materialized, there shouldn't be any cost to creating and ignoring them?

Reply:
Nation and supplier are already listed above, in the same list. These are duplicate entries.